src/ai_commands.js

```diff
@@ -4,45 +4,97 @@ import path from 'path';
 import fs from 'fs';
 import { findFilepath, getIncludeText } from './util';
 
-const configuration = workspace.getConfiguration('autoit');
+const config = Object.assign({}, workspace.getConfiguration('autoit'));
```

Comment:
Copy read-only configuration object into regular object, so we can update it when settings changed by user